migrations/versions/201506051421_4074211727ba_add_indico_unaccent_function.py

```diff
@@ -0,0 +1,37 @@
+"""ADD indico_unaccent function
+
+Revision ID: 4074211727ba
+Revises: 3f3a9554a6da
+Create Date: 2015-06-05 14:21:52.752777
+"""
+
+from indico.core.db.sqlalchemy.core import db
+
+# revision identifiers, used by Alembic.
+revision = '4074211727ba'
+down_revision = '3f3a9554a6da'
+
+
+def upgrade():
+    db.session.execute('''
+        CREATE OR REPLACE FUNCTION indico_unaccent(value TEXT)
```

Comment:
any reason why it would already exist? i'd rather have the migration fail if the database schema is not in sync with alembic's state...